cico_setup.sh

```diff
@@ -83,6 +83,12 @@ function run_tests_without_coverage() {
   echo "CICO: ran tests without coverage"
 }
 
+function run_e2e_tests() {
+  make test-e2e
+  trap cleanup_env EXIT
```

Comment:
You don't need this trap because you have nothing added to the `cleanup_env` function. And so far it looks like you're not needing the cleanup inside.

Reply:
Point taken. Trap to go away.